DependencyInjection/CassandraExtension.php

```diff
@@ -34,11 +34,35 @@ public function load(array $configs, ContainerBuilder $container)
         $this->metadataFactoryLoad($container, $ormConfig);
 
         foreach ($config['connections'] as $connectionId => $connectionConfig) {
+            $emConfig = $this->getEntityManagerConfiguration(
+                $connectionId,
+                $ormConfig['default_entity_manager'],
+                $ormConfig['entity_managers']
+            );
             $connectionConfig['dispatch_events'] = $config['dispatch_events'];
-            $this->ormLoad($container, $connectionId, $connectionConfig);
+            $this->ormLoad($container, $connectionId, $connectionConfig, $emConfig);
         }
     }
 
+    /**
+     * @param string $connectionId
+     * @param string $defaultEmName
+     * @param array  $emConfigs
+     *
+     * @return array
+     */
+    private function getEntityManagerConfiguration($connectionId, $defaultEmName, $emConfigs)
+    {
+        if (!isset($emConfigs[$defaultEmName])) {
```

Comment:
this checking will be executed on every iteration. i think we can move it to outside the loop.

Reply:
I don't see any issue by doing that way.
I made an update to follow your feedback but I am not sure it's better...